.github/workflows/publish.yaml

```diff
@@ -45,6 +45,14 @@ jobs:
           registry-url: 'https://registry.npmjs.org/'
           scope: '@smartbear'
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.2.0
```

Comment:
we should use v3.6.0 as that is the latest and the version we picked in docker-build.yml